packages/components/src/components/hds/side-nav/header/icon-button.hbs

```diff
@@ -2,7 +2,9 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-
+{{!
```

Comment:
I like the idea of adding this to the template 👍

Reply:
Thanks. I copied from someone else :)